docs/userguide/dev/codingguidelines.md

```diff
@@ -445,6 +445,24 @@ type `UBool` must be used in public APIs and in the definition of any persistent
 data structures. `UBool` is guaranteed to be one byte in size and signed; bool is
 not.
 
+Traditionally, ICU4C has defined its own FALSE=0 / TRUE=1 macros for use with UBool.
+Starting with ICU 68 (fall 2020), we no longer define these in public header files
```

Comment:
nit: It might be a bit better to use the term "autumn" instead of "fall", as fall is only for the Northern Hemisphere. :)

Reply:
Good point, but "autumn" has the same problem. Changed to "2020q4".